doc/dash/dash-sonic-hld.md

```diff
@@ -723,85 +726,146 @@ For single endpoint, implmentation shall simply create a sai_dash_tunnel object
-|                       | vnet         |                 | sai_outbound_ca_to_pa_entry_t.dest_vni          | VNET's VNI                                    |
-|                       | ip_address   |                 | sai_outbound_ca_to_pa_entry_t.dip               |                                               |
-|                       |              | routing_type    |                                                 |                                               |
-|                       |              | underlay_ip     | SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_UNDERLAY_DIP   |                                               |
-|                       |              | mac_address     | SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_OVERLAY_DMAC   |                                               |
-|                       |              | metering_bucket | SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_COUNTER_ID     |                                               |
-|                       | vnet*        |                 | sai_pa_validation_entry_t.vnet_id               | VNET's VNI                                    |
-|                       |              | underlay_ip*    | sai_pa_validation_entry_t.sip                   | SAI_PA_VALIDATION_ENTRY_ATTR_ACTION is permit |
-| DASH_ROUTE_RULE_TABLE |              |                 |                                                 |                                               |
-|                       | eni          |                 | sai_inbound_routing_entry_t.eni_id              |                                               |
-|                       | vni          |                 | sai_inbound_routing_entry_t.vni                 |                                               |
-|                       | prefix       |                 | sai_inbound_routing_entry_t.prefix              |                                               |
-|                       |              | action_type     |                                                 |                                               |
-|                       |              | priority        | sai_inbound_routing_entry_t.priority            |                                               |
-|                       |              | protocol        |                                                 |                                               |
-|                       |              | vnet            | SAI_INBOUND_ROUTING_ENTRY_ATTR_SRC_VNET_ID      |                                               |
-|                       |              | pa_validation   | SAI_INBOUND_ROUTING_ENTRY_ATTR_ACTION           | use PA_VALIDATE if true                       |
-|                       |              | metering_bucket |                                                 |                                               |
-
-### 3.2.16 Protobuf encoding
+### 3.2.15 DASH outbound port map table
```

Comment:
Hi @theasianpianist , do you mind to help add the scaling requirements into the HLD as well?

![image](https://github.com/user-attachments/assets/11c4a2fa-50ca-4b29-8183-8dea030f72b3)